lib/t/calls.ex

```diff
@@ -8,10 +8,25 @@ defmodule T.Calls do
   alias T.{Repo, Twilio, Accounts, Feeds, Matches}
   alias T.Calls.{Call, Voicemail}
   alias T.Feeds.{FeedProfile}
-  alias T.Matches.{Match, MatchEvent, MatchContact, Timeslot}
-  alias T.PushNotifications.APNS
+  alias T.Matches.{Match, MatchEvent, MatchContact, Timeslot, ArchivedMatch}
+  alias T.PushNotifications.{APNS, DispatchJob}
   alias T.Bot
 
+  @pubsub T.PubSub
+  @topic "__c"
+
+  defp pubsub_user_topic(user_id) when is_binary(user_id) do
+    @topic <> ":u:" <> String.downcase(user_id)
```

Comment:
This seems to be similar to `Feeds.pubsub_user_topic`?

Reply:
@ruslandoga yes if it simplifies things